LEK SI2Conc edit

LucyKershaw opened the pull request:
Edited LEK code for SI2Conc to include a range for the baseline points to be used.  Test function updated accordingly

LucyKershaw committed:
LEK SI2Conc edit

Edited LEK code for SI2Conc to include a range for the baseline points to be used.  Test function updated accordingly

stadmill commented:
@notZaki Can you please help me understand why this merge is failing?

notZaki committed:
Merge branch 'develop' into LEK_Contribution_edit_SI2Conc

notZaki committed:
Update import

notZaki commented:
@stadmill  This branch was created from a very old version of `develop`. The merge was failing because of one file which was modified in this PR but has also been modified in `develop`, and git can't figure out which version should be kept.
(I think the actual conflict is that the original test file was renamed in `develop`, but this branch is still using an older filename)

I've updated the PR to bring it up to speed with `develop`.
The test still fails, but one thing worth noting is that there already is a similar test in `develop` under `test_SI2Conc_LEK_Edinburgh_UK.py` (underscore between Edinburgh and UK).
It may be better to edit that existing file, and remove the test file that was added in the PR.

stadmill committed:
Delete test/SI_to_Conc/test_SI2Conc_LEK_EdinburghUK.py

Copied edits into develop branch manually, deleting this file to allow the branches to be merged

stadmill commented:
@notZaki Ok I made the changes in the test file manually and pushed to the develop branch. I deleted the test file from Lucy's PR. Some checks are still failing.

notZaki committed:
Use newer API for LEK SI2Conc

notZaki commented:
Should be ok now @stadmill
